### PR TITLE
Feature add: Header and Headline

### PR DESCRIPTION
**Pull request by miguelesco**
-Added:

- Title of headline and the styles
- Descriptions of headline and the styles
- Background Image of the section
- menu icons SVG and styles
- Welcome Message
- content flex

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 15px;
+}
+
+.welcome {
+  font-weight: 600;
+  text-decoration: none;
+  color: black;
```
Comment (Davidosky007) [inline style.css]: i suggest you use the specific colour defined for this section in the template

**Review by Davidosky007 — COMMENTED** [review]
Hi @miguelesco, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you start your commit messages with capital letters this is best practice 
* I can see your project is missing the social link as stated in the design template please 🙏 fix this.
* I suggest you make your PR title descriptive


# _Check the comments under the review._

## Optional suggestions
[OPTIONAL]
- N.A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -10,10 +10,40 @@
...
+                <a href="https://miguelangelescorche.medium.com">
+                    <img src="./assets/icons/medium.svg" alt="twitter">
+                </a>
+                <a href="https://github.com/miguelesco">
+                    <img src="./assets/icons/github.svg" alt="twitter">
+                </a>
+                <a href="">
+                    <img src="./assets/icons/angellist.svg" alt="twitter">
+                </a>
+            </div>
```
Comment (henatan99) [inline index.html:45]: - [ ] OPTIONAL: The social-icons section would have been better wrapped with ul and each a-tag wrapped with li-tag.

**Review by henatan99 — APPROVED** [review]
Hi Miguel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**